src/Views/Errors/403.php

```diff
@@ -3,10 +3,19 @@
  * This View renders a HTTP/1.1 403 Error - for when a <code>AuthUtils::requirePermission()</code> call returns false.
  */
 use \MyRadio\MyRadio\CoreUtils;
+use \MyRadio\MyRadio\URLUtils;
 
 header('HTTP/1.1 403 Forbidden');
 
-CoreUtils::getTemplateObject()->setTemplate('error.twig')
+if ($err_type == 'contract'){
```

Comment:
Not sure I'm a fan of special-casing it like this. Can we perhaps have a switch for the various types of error, defaulting to `error.twig`? Or is there a better solution?

Reply:
Yeah, I wasn't the biggest fan of doing it this way, but it was like 1:30am. 😂 I was thinking afterwards about doing it slightly differently, I'm going to have a look tomorrow.